query-service/src/main/resources/configs/common/application.conf

```diff
@@ -79,7 +79,8 @@ service.config = {
             "EVENT.displaySpanName": "display_span_name",
             "EVENT.errorCount": "error_count",
             "EVENT.exceptionCount": "exception_count",
-            "EVENT.spaceIds": "space_ids"
+            "EVENT.spaceIds": "space_ids",
+            “EVENT.spanCount”: “span_count”
```

Comment:
See comment about naming in config-bootstrapper PR

Reply:
resolved